aula-2/src/main/java/org/acelera/brasil/futebol/Jogador.java

```diff
@@ -0,0 +1,60 @@
+package org.acelera.brasil.futebol;
+
+/**
+ * Classe responsável pela representação de um jogador de futebol.
+ * 
+ * @author edmilson.santana
+ *
+ */
+public class Jogador {
+
+	private final String nome;
+
+	private final String cidade;
+
+	private final String pais;
+
+	private final Posicao posicao;
+
+	private final Integer numeroDeGols;
```

Comment:
o que vc acha se o nome da variável for apenas `gols` como ele já é um inteiro que implícito que estamos falando de número.
O que você acha?

Reply:
Concordo, acho que fica bem melhor chamar a variável de gols